### DISCUSSION TITLE
Connecting Pybamm/liionpack with matlab

### DISCUSSION THREAD

**Discussion by Palash-gaikwad** [pybamm-team/liionpack#252] [General]
Hello team, 
I am new to python, but have good experience with using matlab. I saw your video in advance examples, where you used pybamm libraries in matlab. I want to create a reduced order model of the existing DFN model using DRA teachniques. Can you assist me in how to managed to implement pybamm in matlab environment?

**Comment by TomTranter** [comment]
Here's the code from that example. 
https://github.com/FaradayInstitution/pybamm_simulink_example
N.B It hasn't been updated for a while and we are not supporting it going forward. There will be an alternative way of exporting to matlab available in future as a paid for service. But do let us know if you would like to trial it. @rtimms is working on this

**Reply by Palash-gaikwad** [reply]
Thank you. Yes, I would like a trail. I am working on creating a battery digital twin as part of my master thesis project.

**Reply by TomTranter** [reply]
Sign up to our mailing list here https://ion-works.com/contact
Cheers
